src/main/java/com/vmtapp/enterprise/dao/ITicketDAO.java

```diff
@@ -4,7 +4,7 @@
 
 import java.util.List;
 
-public interface ITicketDao {
+public interface ITicketDAO {
```

Comment:
Dao needs to be all capitalized. Source:https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3